timed/employment/models.py

```diff
@@ -179,3 +181,91 @@ class OvertimeCredit(models.Model):
                                  related_name='overtime_credits')
     date     = models.DateField()
     duration = models.DurationField(blank=True, null=True)
+
+
+class UserAbsenceTypeManager(models.Manager):
+    def with_user(self, user, start_date, end_date):
+        from timed.tracking.models import Absence
+
+        return UserAbsenceType.objects.raw("""
+            SELECT
```

Comment:
I like it this way now... maybe a comment would be good explaining why using raw query. And I think sq1 and sq2 should be renamed for better readability of the SQL (the other aliases are a bit more obvious what they mean).
Otherwise looks great.